test/org/httpkit/ring_request_proxy.clj

```diff
@@ -0,0 +1,71 @@
+(ns org.httpkit.ring-request-proxy
```

Comment:
https://github.com/pubnub/http-kit/blob/6c00b51ee447d1b00ca614c21b54b9208b5b87f9/test/org/httpkit/client_proxy_test.clj#L20 it's being used there